sample/README.md

```
@@ -1,97 +1,344 @@
+### iOS (macOS only)
+- **Xcode 15.0+**
+- **CocoaPods 1.12+**
+- An iOS Simulator or connected physical device
 
-# OR using Yarn
-yarn start
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)